src/components/NavHeader.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <section class="banner">
+    <RouterLink to="/">
+      <img src="../assets/images/white-logo-without-text.png" alt="logo" />
+    </RouterLink>
+    <HamburgerMenu />
+  </section>
+</template>
+
+<script setup>
+import HamburgerMenu from "./HamburgerMenu.vue";
+</script>
+
+<style scoped>
+.banner {
+  height: 10vh;
+  width: 100%;
+  margin-bottom: 2rem;
+  position: relative;
```

Comment:
Do we want the banner to take up different amount of space depending on what kind of screen the user is using? Or do we want to set it to a specific height that stay constant.

Also, why do we use position relative here?